storage/CMakeLists.txt

```diff
@@ -193,6 +193,7 @@ add_library(quickstep_storage_SeparateChainingHashTable ../empty_src.cpp Separat
 add_library(quickstep_storage_SimpleScalarSeparateChainingHashTable
             SimpleScalarSeparateChainingHashTable.cpp
             SimpleScalarSeparateChainingHashTable.hpp)
+add_library(quickstep_storage_SMAIndexSubBlock SMAIndexSubBlock.cpp SMAIndexSubBlock.hpp)
```

Comment:
I guess `quickstep_storage_SMAIndexSubBlock` ranks higher than `quickstep_storage_SimpleScalarSeparateChainingHashTable` regarding the alphabet order.

You could check so in `vim` by selecting all the `add_library` starting with `quickstep_storage_S` and then execute `:sort`.